code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -384,11 +382,10 @@
 	flags = NOBLOODY
 	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 	var/mob/living/creator
-	base_reflectchance = 140
```

Comment:
I am not sure if nerfing this is such a good idea.
This turns the protection of the e-sword into a rng type of situation.

Imho implementing the other changes is sufficient for now.
(Considering that the shield now runs out 3x faster than before and has a reduced power)